Example/Controllers/Extensions/ViewController+MiniApp.swift

```diff
@@ -15,8 +15,11 @@ extension ViewController: MiniAppNavigationDelegate {
                            with responseHandler: @escaping MiniAppNavigationResponseHandler,
                            onClose closeHandler: MiniAppNavigationResponseHandler?) {
         if url.absoluteString.starts(with: "data:") {
+            var base64String = ""
             // currently js sdk is passing no base64 data type
-            let base64String = url.absoluteString.replacingOccurrences(of: "data:;base64,", with: "")
+            if url.absoluteString.contains("data:image/jpeg;base64") {
+                base64String = url.absoluteString.replacingOccurrences(of: "data:image/jpeg;base64", with: "")
```

Comment:
Will this work for other file types other than images? Also, I think the `,` needs to be removed too?

Actually in the case of Android, we just split the string on the `,` and then only decode the portion afterwards. Would that work here?

Reply:
Done